src/postprocessing.py

```diff
@@ -0,0 +1,111 @@
+
+import os
+import pandas as pd
+from metriq import MetriqClient
+from metriq.models.result import ResultCreateRequest
+from metriq.models.submission import (Submission, SubmissionCreateRequest)
+
+METRIQ_TOKEN = os.getenv("METRIQ_TOKEN")
+RESULTS_PATH = os.path.abspath(os.path.join( os.path.dirname( __file__ ),"..", "benchmarking", "results"))
+
+CONTENT_URL = "https://github.com/qiskit-community/submit-metriq"
+THUMBNAIL_URL = "https://avatars.githubusercontent.com/u/30696987?s=200&v=4"
+
+# Metriq API parameters and associated ids
+METHOD = {"8": "Qiskit compilation"}
+PLATFORMS = {"64": "Rigetti 16Q Aspen-1 ", "69": "ibmq-rochester"}
+TAGS = ["quantum circuits", "compiler", "compilation", "ibm qiskit"]
+TASKS = {"25": "ex1_226.qasm", "26": "ex1_226.qasm (Aspen)", "27": "ex1_226.qasm (Rochester)"}
+
+def get_id(param_list: dict, param_name: str) -> str:
+  # Return key from value
+  return list(param_list.keys())[list(param_list.values()).index(param_name)]
+
+def get_platform_id(keywork: str) -> str:
+  norm_keyword = keywork.lower()
+  for key, value in PLATFORMS.items():
+    norm_value = value.lower()
+    if norm_keyword in norm_value:
+      return key
+  return None
+
+# TODO Keep track of results already submitted and prepare automation pipeline
+def submit_all(task_name: str, submission_id: str = None):
+  client = MetriqClient(token=METRIQ_TOKEN)
+
+  if not submission_id:
+    create_new_submission(client,task_name)
+
+  # Populate submission parameters
+  # Task
+  task_id = get_id(TASKS, task_name)
+  client.submission_add_task(submission_id, task_id)
+
+  # Method
+  method_id = get_id(METHOD, "Qiskit compilation")
+  client.submission_add_method(submission_id, method_id)
+
+  # Tags
+  for tag in TAGS:
+    client.submission_add_tag(submission_id, tag)
+
+  # TODO Find a way to to update params below using the API
+  # submission.codeUrl
+  # submission.platform
```

Comment:
@vprusso @WrathfulSpatula 
Did I miss something or I just couldn't find a way to add these fields to my submission? 
Maybe they could be added when creating a new `SubmissionCreateRequest` item.

Reply:
Opened an issue for this: https://github.com/unitaryfund/metriq-client/issues/108

Reply:
You are definitely correct here, @karlaspuldaro . Thank you for opening that issue. I've put up a few PRs that (I hope) resolve the points you brought up. Thank you again for taking the time to write those out, and I hope that these fixes work. If not, please let us know!